src/test/java/teammates/test/cases/ui/pagedata/FeedbackSubmissionEditPageDataTest.java

```diff
@@ -0,0 +1,140 @@
+    private StudentAttributes student;
+    
+    private FeedbackSessionAttributes feedbackSession;
+    private FeedbackQuestionAttributes question;
+    private List<FeedbackResponseAttributes> responses = new ArrayList<FeedbackResponseAttributes>();
+    
+    private Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> questionResponseBundle = 
+                                    new HashMap<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>();
+    
+    private Map<String, Map<String, String>> recipientList = new HashMap<String, Map<String, String>>();
+    private Map<String,String> recipients = new HashMap<String,String>();
+    
+    @BeforeClass
+    public static void classSetUp() throws Exception {
+        printTestClassHeader();
+    }
+    
+    public void createData(String studentKey) {
+        studentAccount = dataBundle.accounts.get("student1InCourse1");
+        student = dataBundle.students.get(studentKey);
+        
+        pageData = new FeedbackSubmissionEditPageData(studentAccount, student);
+        
+        feedbackSession = dataBundle.feedbackSessions.get("session1InCourse1");
+        question = dataBundle.feedbackQuestions.get("qn1InSession1InCourse1");      
+        
+        responses.add(dataBundle.feedbackResponses.get("response1ForQ1S1C1"));
+        responses.add(dataBundle.feedbackResponses.get("response2ForQ1S1C1"));      
+        
+        questionResponseBundle.put(question, responses);      
+        
+        recipients.put(student.email, Const.USER_NAME_FOR_SELF);
+        recipientList.put(question.getId(), recipients);
+        
+        pageData.bundle = new FeedbackSessionQuestionsBundle(feedbackSession, questionResponseBundle, recipientList);
+        pageData.bundle.questionResponseBundle.put(question, responses);
+    }   
+    
+    @Test
+    public void testAll() {
+        ______TS("test typical case");       
+        createData("student1InCourse1");         
+        
+        pageData.init(student.key, student.email, student.course);
+        
+        assertEquals("You are submitting feedback as <span class='text-danger text-bold text-large'>"
+                      + "student1 In Course1</span>. You may submit feedback and view results without logging "
+                      + "in. To access other features you need <a href='/page/studentCourseJoinAuthentication?"
+                      + "studentemail=student1InCourse1%40gmail.tmt&courseid=idOfTypicalCourse1' class='link'>"
+                      + "to login using a google account</a> (recommended).", pageData.getRegisterMessage());
+        
+        assertEquals("/page/studentFeedbackSubmissionEditSave", pageData.getSubmitAction());
+        assertEquals("/page/studentFeedbackQuestionSubmissionEditSave", pageData.getSubmitActionQuestion());
+        assertFalse(pageData.isModeration());
+        assertFalse(pageData.isSessionOpenForSubmission());
+        assertFalse(pageData.isSubmittable());
```

Comment:
Considering these 5 lines are repeated 3 times, would it be better to extract them into another method?

Also, are the tests sufficient?
Is there any need to test for other values (`true`?) from the `is_____()` methods?